packer-scripts/remove-default-users

```diff
@@ -22,10 +22,7 @@ __remove_user() {
     return
   fi
 
-  ps -Fwww -u "${u}" || echo "Suppressing ps exit $?"
-  killall -TERM -w -v -u "${u}" || echo "Suppressing killall exit $?"
```

Comment:
Should this be removed? The `userdel` docs on my machine seems to imply that `userdel -f` will delete the user even if they are logged in, but won't terminate any of their processes.